browser/js/content.js

```diff
@@ -105,11 +150,59 @@ function submitChat(e) {
   });
   port.onMessage.addListener(function(response) {
     console.log("response: ", response);
+
+    if (response.status != "ok") {
+      renderError(f, response.message);
+      submitButton.value = "Error";
+      return;
+    }
+
+    removeChat(f, true /* skipCheck */);
+    nudgeCallback();
   });
+}
+
+// Render error message inside our chat widget.
+function renderError(chatForm, msg) {
+  const p = document.createElement("p");
+  p.className = "keybase-error";
+  p.innerText = msg;
+  chatForm.appendChild(p);
+}
+
+// Post a Reddit thread reply on the given comment node.
+function postReply(commentNode, text) {
+  // This will break if there is no reply button.
+  const commentID = commentNode.getAttribute("data-fullname");
+  const replyLink = commentNode.getElementsByClassName("reply-button")[0].firstChild;
```

Comment:
we should try and detect this and fail instead of just error out due to a referenced null. can use try / catch